frontend/src/features/leases/detail/LeasePages/LeasePageForm.tsx

```diff
@@ -32,7 +33,9 @@ export const LeasePageForm: React.FunctionComponent<ILeasePageFormProps> = ({
           ...lease,
         }}
         enableReinitialize={true}
-        onSubmit={noop}
+        onSubmit={() => {
+          refreshLease();
+        }}
```

Comment:
not sure if there is a cleaner way to refresh the parent container.

Reply:
a common pattern is to use the return value from the add/update to update the lease being displayed.

Reply:
saves a round trip.

Reply:
I was thinking about that, but I thought it would be better to get the entire tree that renders this component since we don't have each page acting independently. like /leases/65/insurances vs  /leases/65/pages=insurances

Reply:
Yes, actually my PR will be addressing that, and moving us to using a sub-router for the lease pages.